test/Passes/pass-map-to-brgemm.mlir

```diff
@@ -33,3 +33,30 @@ func.func @blocked_matmul(%arg0: tensor<4x16x32x32xf32>, %arg1: tensor<8x16x32x3
     } -> tensor<4x8x32x32xf32>
   return %1 :  tensor<4x8x32x32xf32>
 }
+
```

Comment:
Don't forget to add the tests requested in https://github.com/plaidml/tpp-mlir/pull/329#issuecomment-1430812339